Cargo.toml

```diff
@@ -95,6 +95,12 @@ near-token = { version = "0.3", features = [
 ] }
 
 near-abi = "0.4.2"
+near-workspaces = "0.19.0"
+near-verify-rs = "0.3.0"
+camino = "1.1.1"
+git2 = "0.19"
+tempfile = "3.10.1"
```

Comment:
all of the new deps have to be feature-gated with e.g. `verify_contract` feature, so that `cargo-near` can pull in `near-cli` as a dependency without this new feature, in order to not have e.g. potentially 2 versions of `near-verify-rs` in its dependency tree. 

strictly speaking, `cargo-near` depends only on `deploy-contract` feature of `near-cli` (doesn't exist), but that's definetely not a top priority to try to structure dependencies so, that only things needed for `deploy-contract` are imported by `cargo-near`

Reply:
Done